README.md

```diff
@@ -85,14 +85,18 @@ Or, if you're compiling and running using Gradle:
 
 `$ gradlew run --args="-cli path/to/config"`
 
-Finally, you can activate a special `convert-to-cdf` function via the command line to export the CVR as a NIST common data format (CDF) .json instead of tabulating the results, e.g.:
+You can also activate a special `convert-to-cdf` function via the command line to export the CVR as a NIST common data format (CDF) .json instead of tabulating the results, e.g.:
 
 `$ rcv -cli path/to/config convert-to-cdf`
 
 Or, again, if you're compiling and running using Gradle:
 
 `$ gradlew run --args="-cli path/to/config  convert-to-cdf"`
 
+Finally, there's a `convert-dominion-to-generic-csv` option as well for converting Dominion JSON CVRs into a standardized, generic .csv format:
```

Comment:
Perhaps here (and above) should mention where the output file will be located.

Reply:
I don't think it's necessary because it says clearly where it outputs when you run it (and I anticipate this changing and don't want to have to remember to update the docs).